root/inc/dependencies-graph.html

```diff
@@ -12,11 +12,13 @@ <h3 class="text-center"><% release.name %></h3>
 </div>
 
 <script>
-    $('#dependencies-graph').on('show.bs.modal', function() {
-        var iframe = $("iframe", this);
-        var base   = "//widgets.stratopan.com/wheel";
-        var src    = base + "?q=<% release.name | uri %>";
-        iframe.attr("src") || iframe.attr("src", src);
-        if (!iframe) return e.preventDefault();
+    document.addEventListener('DOMContentLoaded', function () {
+        $('#dependencies-graph').on('show.bs.modal', function() {
+            var iframe = $("iframe", this);
+            var base   = "//widgets.stratopan.com/wheel";
+            var src    = base + "?q=<% release.name | uri %>";
+            iframe.attr("src") || iframe.attr("src", src);
+            if (!iframe) return e.preventDefault();
+        });
```

Comment:
We've switched the dep graph over to https://cpandeps.grinnz.com/?dist=Moose, so we can probably just remove this?